packages/core/src/services/evaluationsV2/update.ts

```diff
@@ -93,9 +93,13 @@ export async function updateEvaluationV2<
       versionId: result.id,
     } as unknown as EvaluationV2<T, M>
 
-    await pingProjectUpdate({ projectId: commit.projectId }, tx).then((r) =>
```

Comment:
do not write side effects in services instead publish an event and let handlers handle side effects